modules/reference/pages/images/images-image-list.adoc

```diff
@@ -1,33 +1,26 @@
 [[ref.webui.images.list]]
 = Image List
 
-If you click menu:Main Menu[Images > Images], an overview listing of your images appears.
-Several columns provide information about each image:
 
+The menu:Images[Image List] section displays your current operating system images.
 
-image::images_list_images.png[scaledwidth=80%]
+Click btn:[Import] to import a new image.
+Click btn:[Refresh] to update the list.
 
+Perform bulk actions by checking images in the list.
+Click btn:[Delete] to bulk delete images.
 
-* Select box: To select images, mark the appropriate check boxes. Selected images can be deleted simultaneously via the btn:[Delete] button that appears in the upper right corner while selecting images.
-* [guimenu]``Name``:
-* [guimenu]``Version`` and [guimenu]``Revision``:
-* [guimenu]``Updates``: Shows which type of update action is applicable to the image or confirms that the image is up-to-date.
+[[image-list-columns]]
+[cols="1,1", options="header"]
+.Image List Columns
+|===
+| Column               | Description
+| Name                 | Name of the image.
+| Version and Revision | Version and revision of the image.
+| Updates              | Any updates that are currently available for the image.
+| Patches and Packages | Any patches or packages that are currently available for the image.
+| Build                | The time and date the image was built.
```

Comment:
Unless there is some change in uyuni, this does not inform about when the image was built, but rather if it is build, scheduled, building or failed.